pkg/kubelet/kuberuntime/kuberuntime_container_windows.go

```diff
@@ -76,7 +77,7 @@ func (m *kubeGenericRuntimeManager) generateWindowsContainerConfig(container *v1
 		wc.Resources.CpuMaximum = cpuMaximum
 	}
 
-	cpuShares := milliCPUToShares(cpuLimit.MilliValue(), isolatedByHyperv)
+	cpuShares := milliCPUToShares(cpuLimitMillValue, isolatedByHyperv)
 	if cpuShares == 0 {
```

Comment:
Are you seeing any warnings from Docker in this case? I think that if `HostConfig.CpuShares` is passed to Docker, that still takes precedence over NanoCpus.

Reply:
Yeap,It seems that cannot assign values ​​to both parameters at the same time


![image](https://user-images.githubusercontent.com/7907809/70588738-b0380980-1c08-11ea-819b-24c839e102b0.png)